File: dashboard/Domain/entities/order.py
class Order:
    def __init__(self, orderUuid, store_uuid, user_uuid, status, total_price, created_at, updated_at):
        self.orderUuid = orderUuid
        self.store_uuid = store_uuid
        self.user_uuid = user_uuid
        self.status = status
        self.total_price = total_price
        self.created_at = created_at
        self.updated_at = updated_at

File: src/Order/Domain/Entities/Order.ts
import { OrderItem } from "./OrderItem";

export class Order {
    id: string;
    user_uuid: string;
    store_uuid: string;
    status: string;
    total_price: number;
    created_at: Date;
    updated_at: Date;
    items: OrderItem[];

    constructor(
        id: string,
        user_uuid: string,
        store_uuid: string,
        status: string,
        total_price: number,
        created_at: Date,
        updated_at: Date,
        items: OrderItem[]
    ) {
        this.id = id;
        this.user_uuid = user_uuid;
        this.store_uuid = store_uuid;
        this.status = status;
        this.total_price = total_price;
        this.created_at = created_at;
        this.updated_at = updated_at;
        this.items = items;
    }
}
